dist/tools/travis-scripts/build_and_test.sh

```diff
@@ -7,6 +7,8 @@
 # directory for more details.
 #
 
+CI_BASE_BRANCH=${CI_BASE_BRANCH:-master}
```

Comment:
optional, that's why I only say it after ACK: wouldn't an `export` make sense here?